gwt/launchers/CodeServer.html

```diff
@@ -91,81 +76,52 @@ <h1 id="title">GWT Plugins Documentation</h1>
    <div id="middle"> 
     <div id="tpl-content"> 
      <div> 
-      <div class="toc-page" style="display:block;">
+      <div class="toc-page" style="display:none;">
```

Comment:
maybe remove it? it doesnt look helpful on this page at least.

ditto others in this review

Reply:
Okay - I'm not sure what exactly is generating it or now, or where the other information is from. Given that it is generated, could/should we generate as part of the build, rather than check them in...?

I'm happy to have @protoism sign off if he understands, i'm just providing my two cents on here.

Reply:
Sorry, i do not understand that because of my bad english. :-(
The pages are generated by the existing "mvn package" command as stated in the master branch.
This branch contains the actual generated pages, that are visible on the documentation page.
Everything that i changed has been changed on the master branch by the other pull request.
I've now regenerated the pages with english locale to fix the spelling of February on all pages.

Reply:
Got it, thank you - I likewise should have looked more carefully and seen the `gh-pages` branch.